src/nerve_tools/clean_nerve.py

```diff
@@ -63,33 +63,27 @@ def clean(
     removed_count = 0
     services: Iterable[str] = []
 
-    try:
```

Comment:
I wonder if we really need this script anymore at all. Funnily enough, we do have a problem with orphan nodes but not exactly the ones described here (they aren't missing data/owner).

Given that it's not been cleaning up in the right place for so long, I'm tempted to say we just remove it. Or at least we ensure that puppet isn't running it.

Reply:
Filed #65